arviz/plots/backends/matplotlib/elpdplot.py

```diff
@@ -36,7 +36,6 @@ def plot_elpd(
         **backend_kwarg_defaults(),
         **backend_kwargs,
     }
-    backend_kwargs.setdefault("constrained_layout", not xlabels)
```

Comment:
Not sure this has to be removed. IIRC, I added this because when `xlabels=True` `plot_elpd` uses the `fig.autofmt_time` matplotlib helper function which is not compatible with constrained layout, so that aimed to override the style defaults unless set explicitly.